apps/nextjs-website/src/messages/it.json

```diff
@@ -72,5 +72,27 @@
             "subscribedLabel": "Iscrizione confermata",
             "unsubscribeLabel": "Annulla iscrizione"
         }
+    },
+    "errors": {
+        "CodeDeliveryFailureException": "Impossibile inviare il codice di verifica.",
```

Comment:
Is there a reason why you used PascalCase naming convention instead of camelCase?

Reply:
These are the error codes that we receive from Cognito and they are PascalCase, we can't change that